docs/how-to/hip_runtime_api/multi_device.rst

```diff
@@ -0,0 +1,287 @@
+.. meta::
+    :description: This chapter describes how to use multiple devices on one host.
+    :keywords: ROCm, HIP, multi-device, multiple, GPUs, devices
+
+.. _multi-device:
+
+*******************************************************************************
+Multi-device management
+*******************************************************************************
+
+Device enumeration
+===============================================================================
+
+Device enumeration involves identifying all the available GPUs connected to the
+host system. A single host machine can have multiple GPUs, each with its own 
+unique identifier. By listing these devices, you can decide which GPU to use
+for computation. The host queries the system to count and list all connected
+GPUs, ensuring that the application can leverage the full computational power
```

Comment:
Is the recommendation to have an application that lists the devices and their properties so that for a given machine you can understand how to deploy the application in the future? Or is the host code intended to evaluate the code on some basis and make a selection during runtime? I am just trying to understand the recommendation to the user?

Reply:
Nice catch!
Typically during deployment planning. I added a sentence to clarify it.